server/neptune/workflows/activities/main.go

```diff
@@ -29,9 +30,9 @@ const (
 	// binDirName is the name of the directory inside our data dir where
 	// we download binaries.
 	BinDirName = "bin"
-	// terraformPluginCacheDir is the name of the dir inside our data dir
-	// where we tell terraform to cache plugins and modules.
-	TerraformPluginCacheDirName = "plugin-cache"
+	// PluginCacheDir is the name of the dir inside our data dir
+	// where we tell commands to cache plugins and modules.
+	PluginCacheDirName = "plugin-cache"
```

Comment:
Did you end to change these? This is terraform specific so why not just keep that in the name.

Reply:
good callout, will revert